Fix issue #46 - Keyboard backlight remains off after sleep on Catalina

AR-CADE opened the pull request.

AR-CADE committed:
Fix issue #46 - Keyboard backlight remains off after sleep on Catalina

AR-CADE committed:
removed useless code

AR-CADE committed:
re-implemented the last commit since it can be useful  not only for sleep but also for reboot and shutdown

hieplpvip commented on AsusSMC/AsusSMC.cpp:
```diff
@@ -163,6 +163,38 @@ OSDictionary *AsusSMC::getDictByUUID(const char *guid) {
+    } else {
+        DBGLOG("atk", "Waking up");
+        kbl_level = readKBBacklightFromNVRAM();
+        setKBLLevel(kbl_level);
```
Please change this to `setKBLLevel(kbl_level, false, false);`

AR-CADE replied:
done, 

also fix a copy-paste mistake that I made on power off, the level is now set explicitly to 0

hieplpvip commented on AsusSMC/AsusSMC.cpp:
```diff
@@ -163,6 +163,38 @@ OSDictionary *AsusSMC::getDictByUUID(const char *guid) {
+};
+
+IOReturn AsusSMC::setPowerState(unsigned long powerStateOrdinal, IOService * whatDevice) {
+
```
Please delete this blank line

AR-CADE replied:
done

hieplpvip reviewed (changes requested):
Looks good to me. There are 2 small problems, please fix them and I'll merge this. Thanks very much.

AR-CADE committed:
- Fix a mistake on power off , the baklight level should be set to 0 without saving to nvram nor showing the badge nor even set the kbl_level variable, just like in the first commit e9dc2cd
- On waking up, the baklight level is now set without saving to nvram nor showing the badge as advised by hieplpvip
- Fix a typo advised by hieplpvip
